### DISCUSSION TITLE
Is there a way to put an object in a model?

### DISCUSSION THREAD

**Discussion by milosworks** [prisma/prisma#11953] [Q&A] [answered]
Im new at prisma, but i was thinking if it possible to get data like this

```
{
  id: 1.
  name: 'Vyrek',
  adress: {
    country: 'Mexico',
    ...
  }
}
```

without making a relation, i dont want another collection just for the same topic as the main model, like this

```
model User {
  id String @id @map("_id")
  name String
  adress {
    country String
  }
}
```

**Comment by nurul3101** [comment, marked as answer]
Hey @VyrekXD 👋🏻 ,
Unfortunately that is not possible, to have data in a nested format you would need to have a different collection.
In your case it would be `address` collection and while fetching data you can fetch address while querying users.
Your query could look like
```javascript
await prisma.user.findUnique({
  where: {
    id: id,
  },
  include: {
    address: true,
  },
});
```
Here is a link to include [reference](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#include)

Please let me know if you face any other issues.

**Reply by milosworks** [reply]
Is it planned for a future version?

**Reply by nurul3101** [reply]
Embedded Document Support is actively being worked upon, You can follow along with the related issues to monitor progress, Also Please leave a 👍🏻 to the issue which is related to your use case, and a comment explaining your exact needs would help our product and engineering team in prioritising it.

- https://github.com/prisma/prisma/issues/8837
- https://github.com/prisma/prisma/issues/2505

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
